truffle-config.js

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
Do we need this file instead of truffle.js?

Reply:
It's how `truffle init` generates a project now:
https://truffleframework.com/docs/truffle/reference/configuration